src/main/java/org/jmhsrobotics/frc2025/subsystems/led/RedLEDCommand.java

```diff
@@ -0,0 +1,19 @@
+package org.jmhsrobotics.frc2025.subsystems.led;
+
+import edu.wpi.first.wpilibj.LEDPattern;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.Command;
+
+public class RedLEDCommand extends Command {
+  private LED led;
+  private final LEDPattern red = LEDPattern.solid(Color.kRed);
+
+  public RedLEDCommand(LED led) {
+    this.led = led;
+  }
+
+  @Override
+  public void execute() {
+    led.setPattern(red);
+  }
```

Comment:
This command probably should be changed so you can create any color by passing it as a constructor. Not a big deal as I suspect things will change once we layout the strips on the robot.